src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -60,14 +61,24 @@ public static <T> Logger getLogger(Class<T> clazz) {
     }
 
     /**
-     * Adds the {@code consoleHandler} to the {@code logger}. <br>
-     * Creates the {@code consoleHandler} if it is null.
+     * Creates and returns a base logger with the default configuration with the name {@code BASE_LOGGER_NAME}.
+     * The base logger is the parent of all other loggers. Changing the level of the base logger will change
+     * the level of all other loggers if they are untweaked. Similarly, all loggers created from this class will
+     * inherit the handlers of the base logger.
      */
-    private static void addConsoleHandler(Logger logger) {
-        if (consoleHandler == null) {
-            consoleHandler = createConsoleHandler();
+    private static Logger getBaseLogger() {
+        Logger baseLogger = Logger.getLogger(BASE_LOGGER_NAME);
+        baseLogger.setUseParentHandlers(false);
+        removeHandlers(baseLogger);
+        baseLogger.addHandler(createConsoleHandler(Level.ALL));
```

Comment:
Perhaps add a note about it. Something like this?
```suggestion
        // Level.ALL is used because loggers filter the log messages by level already,
        // there is no need to control log message level of the handlers 
        baseLogger.addHandler(createConsoleHandler(Level.ALL));
```